tests/manage/cluster/cluster_expansion/test_add_capacity.py

```diff
@@ -35,3 +36,29 @@ def test_add_capacity(self):
         else:
             pytest.skip("UPI not yet supported")
         # ToDo run IOs
+
+    @pytest.mark.parametrize(
```

Comment:
@fmarchioni Since most of the functionality already implemented in lines 22-37, please try to aggregate the new test case as a param under the existing one

Reply:
Thanks for your review @ebenahar . I've aggregated the new test into the existing one. If you are ok with the change I'll squash commits so that you can merge it.